xtask/src/main.rs

```diff
@@ -9,17 +9,18 @@
 //! `.cargo/config`.
 mod help;
 
+use autocfg;
 use core::fmt::Write;
 use core::str;
 use pico_args::Arguments;
 use std::{env, path::PathBuf};
 use xtask::{
     codegen::{self, Mode},
-    install_format_hook, run, run_clippy, run_fuzzer, run_rustfmt, run_with_output, Cmd, Result,
+    install_format_hook, run, run_clippy, run_fuzzer, run_rustfmt, Cmd, Result,
 };
 
 // Latest stable, feel free to send a PR if this lags behind.
-const REQUIRED_RUST_VERSION: u32 = 39;
+const REQUIRED_RUST_VERSION: (usize, usize) = (1, 39);
```

Comment:
Horrible idea `: f32 = 1.39` (No, we won’t do this)

Reply:
`1 * 1_000 + 39` :laughing: